function/fetchFile.js

```diff
@@ -46,6 +46,7 @@ function makePrediction(file,url) {
       headers:{
         'Content-Type': 'application/json'
       }
+      return response;
```

Comment:
No need here - the response is returned to the "on resolve" (on success) handler, which is represented by "then", just below.
---
In short - a "Promise" (a promise to provide a result) can either "resolve" or "reject". Resolve -> calls `then`, and reject -> calls `catch` (if defined).

Future reading: https://javascript.info/promise-chaining

Reply:
Ohhh I understand! That makes a lot of sense, thank you!